src/app/dynamic-forms/cinchy-dynamic-forms.component.ts

```diff
@@ -554,7 +554,9 @@ export class CinchyDynamicFormsComponent implements OnInit, OnChanges, OnDestroy
           let cinchyIdForMatchFromParentResp = (await this._cinchyService.executeCsql(queryToGetMatchIdFromParent, null, null, QueryType.DRAFT_QUERY).toPromise()).queryResult.toObjectArray();
           let idForParentMatch = cinchyIdForMatchFromParentResp[0]['idParent'];
           if (idForParentMatch) {
-            selectQuery.query = selectQuery.query.replace('@parentCinchyIdMatch', idForParentMatch);
+             selectQuery.params = {
+              '@parentCinchyIdMatch': idForParentMatch
+            };
```

Comment:
change to something like this so that we're not replacing the entire .params property with a new object

if (selectQuery.params == null) {
	selectQuery.params = {};
}
selectQuery.params['@parentCinchyIdMatch'] = idForParentMatch;